cpp/include/copying.hpp

```diff
@@ -38,10 +38,11 @@ gdf_column empty_like(gdf_column const& input);
  * @brief Allocates a new column of the same size and type as the input.
  *
  * @param input The input column to emulate
+ * @param allocate_mask Optional whether or not to allocate bitmask
  * @param stream Optional stream in which to perform copies
  * @return gdf_column An allocated column of same size and type of input
  */
-gdf_column allocate_like(gdf_column const& input, cudaStream_t stream = 0);
+gdf_column allocate_like(gdf_column const& input, bool allocate_mask = true, cudaStream_t stream = 0);
```

Comment:
Adding the `allocate_mask` parameter is counter to the name of this function. The intent is to allocate a column that is "like" the `input`, meaning, only allocate a mask if the input has one.

Reply:
So perhaps another function `allocate_like_no_mask`?

Reply:
Where do you need this? When do you allocate a new column without a mask from an existing one with a mask?

Reply:
See below conversations - this was resolved